guides/common/modules/proc_registering-clients-using-the-bootstrap-script.adoc

```diff
@@ -28,6 +28,7 @@ For more information, see {ManagingHostsDocURL}Registering_Hosts_by_Using_the_Bo
 <2> Include the `--force` option to register the client that has been previously registered to a standalone {SmartProxy}.
 <3> Include the `--puppet-ca-port 8141` option if you use Puppet.
 
+ifdef::satellite[]
```

Comment:
Note to self: I found out this whole procedure is only included in Satellite and Orcharhino so this probably isn't needed at all.